src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -1,51 +1,68 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
 import DropdownBubble from 'app/components/dropdownBubble';
-import DropdownButton from 'app/components/dropdownButton';
-import DropdownMenu from 'app/components/dropdownMenu';
+import DropdownButton, {
+  Props as DropdownButtonProps,
+} from 'app/components/dropdownButton';
+import DropdownMenu, {GetActorPropsFn} from 'app/components/dropdownMenu';
 import MenuItem from 'app/components/menuItem';
+import theme from 'app/utils/theme';
+
+type DefaultProps = {
+  /**
+   * Should the menu contents always be rendered?  Defaults to true.
+   * Set to false to have menu contents removed from the DOM on close.
+   */
+  alwaysRenderMenu: boolean;
+  /**
+   * Width of the menu. Defaults to 100% of the button width.
+   */
+  menuWidth: string;
+};
+
+type Props = DefaultProps & {
+  /**
+   * String or element for the button contents.
+   */
+  label?: string | JSX.Element | null;
```

Comment:
I'm not too familiar with `JSX.Element. 

`React.ReactNode` may be more suitable as the type for `label`.